client/src/app/app.component.ts

```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title = 'empty-cli-project';
```

Comment:
You can just delete this. The title variable isn't being referenced in the component template.